cmd/kubeadm/app/cmd/config.go

```diff
@@ -206,8 +207,74 @@ func uploadConfiguration(client clientset.Interface, cfgPath string, defaultcfg
 	return uploadconfig.UploadConfiguration(internalcfg, client)
 }
 
-// NewCmdConfigListImages returns the "kubeadm images" command
-func NewCmdConfigListImages(out io.Writer) *cobra.Command {
+// NewCmdConfigImages returns the "config images" command
+func NewCmdConfigImages(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "images",
+		Short: "Interact with container images used by kubeadm.",
+		RunE:  cmdutil.SubCmdRunE("images"),
+	}
```

Comment:
did these two PRs overlap?
https://github.com/kubernetes/kubernetes/pull/63811

oddly, for the other one the bots are saying `PR is unable to be automatically merged. Needs rebase.` but github thinks there is no need for rebase.

the refactor and `pull` addition could have been separate commits.

Reply:
Yeah, that was me basing this branch off that one. I forgot to mention that, my mistake. The refactor and the addition are in two separate commits. I just fixed the merge conflict, it should look a lot better now.